x-pack/plugins/index_management/public/services/sort_table.js

```diff
@@ -17,7 +17,14 @@ const unitMagnitude = {
 };
 const byteSort = (fieldName) => (item) => {
   const rawValue = item[fieldName];
-  const [ , number, unit] = rawValue.match(/(.*)([kmgtp]b)/);
+  if (!rawValue) {
```

Comment:
Might be worth adding a comment that closed indices will be missing the `rawValue` (or whatever the case may be that causes closed indices to cause errors here).

Reply:
Makes sense, done.